pkgs/games/openrct2/default.nix

```diff
@@ -25,51 +25,54 @@
 , pkg-config
 , speexdsp
 , zlib
-}:
-
-let
-  openrct2-version = "0.4.4";
-
   # Those versions MUST match the pinned versions within the CMakeLists.txt
   # file. The REPLAYS repository from the CMakeLists.txt is not necessary.
-  objects-version = "1.3.8";
-  openmsx-version = "1.1.0";
-  opensfx-version = "1.0.2";
-  title-sequences-version = "0.4.0";
+, openrct2-version ? "0.4.4"
+, openrct2-sha256 ? "sha256-kCGX+L3bXAG9fUjv04T9wV+R20kmmuREHY8h0w+CESg="
+, objects-version ? "1.3.8"
+, objects-sha256 ? "sha256-7fKv2dSsWJ/YIneyVeuPMjdNI/kgJ7zkMoAgV/s240w="
+, openmsx-version ? "1.1.0"
+, openmsx-sha256 ? "sha256-SqTYJSst1tgVot/c4seuPQVoxnqWiM2Jb/pP3mHtkKs="
+, opensfx-version ? "1.0.2"
+, opensfx-sha256 ? "sha256-AMuCpq1Hszi2Vikto/cX9g81LwBDskaRMTLxNzU0/Gk="
+, title-sequences-version ? "0.4.0"
+, title-sequences-sha256 ? "sha256-anqCZkhYoaxPu3MYCYSsFFngOmPp2wnx2MGb0hj6W5U="
```

Comment:
This is not that nice, could we use srcs instead? Or do you have another idea how we could use override on the sources rather?

Reply:
Ah right, yeah that would be better :) Let me get back to that :)

Reply:
Hmm, I think I spoke to soon. The version numbers are used separately too, to check if the specified version of the dependency is correct; so using srcs here would still require separate version numbers unless we refactor that check too.

What would be your preferred way of doing this?

Reply:
hey both, sorry for the late reply, I was on holiday :)

I agree accepting the sources would be way better, I clearly really needed that holiday... Sorry for the waste of time, I'll get back to it :)